server-common/src/main/java/org/apache/gravitino/server/web/JettyServer.java

```diff
@@ -260,6 +260,29 @@ private void initializeWebAppServletContextHandler() {
     servletContextHandler = new WebAppContext();
 
     boolean isUnitTest = System.getenv("GRAVITINO_TEST") != null;
+    if (isUnitTest) {
```

Comment:
I'm curious about this change, you relocate the jetty package not exclude it, why couldn't find the jetty class?

Reply:
After remove `server-common` test dependences in other modules, this seems unnecessary anymore?

Reply:
@xunliu need answer.

Reply:
Why does only UT need this?